package.json

```diff
@@ -121,17 +119,17 @@
     "conventional-github-releaser": "3.1.5",
     "cypress": "13.13.0",
     "detox": "20.18.5",
-    "eslint": "8.57.0",
+    "eslint": "^8.19.0",
```

Comment:
Was downgrading this package a mistake?

Reply:
Even `typescript`.

Reply:
I used those versions from the React Native sample app to avoid potential compatibility issues.

Reply:
Update: I noticed that `eslint` was resolving to `8.57.1`. Therefore, I've updated it to that version.